edx-platform/pearson-vue-theme/lms/templates/index.html

```diff
@@ -8,7 +8,7 @@
 <div class="background-login"></div>
 
 <div class="login-container">
-  <div>
+  <div class="signin-logo-container">
```

Comment:
I think you can apply the same styles of "signin-logo-container" to "signin-logo" and get rid of "signin-logo-container". It's just to avoid unnecessary divs... @Jacatove

Reply:
I agree with you. So, I have proceeded as you suggested. @Squirrel18
I also edited the size of the logo for some resolutions to improve how they look in terms of the expected behavior.